### PR TITLE
Adding locales/en.json key value translation file

### PR DESCRIPTION
**Pull request by sg00dwin**

### CODE REVIEW

**Review by jhadvig — CHANGES_REQUESTED** [review]
@sg00dwin thanks for the PR. Could you please move the en locales to the the locales_en block in the [locales.go](https://github.com/jhadvig/oauth-server/blob/CONSOLE-2535/pkg/server/login/locales.go#L40). Also who will be adding the resot of the locales ? Meaning zh, ko, jp ?
We also need to update the [default template](https://github.com/jhadvig/oauth-server/blob/CONSOLE-2535/pkg/server/login/templates.go) so it contains the go macros.
Could you also link the oauth-template PR.
THanks ! :)

**Comment by sg00dwin** [comment]
> @sg00dwin thanks for the PR. Could you please move the en locales to the the locales_en block in the [locales.go](https://github.com/jhadvig/oauth-server/blob/CONSOLE-2535/pkg/server/login/locales.go#L40). 

Moved strings to locales.go 
Is this correct?

> Also who will be adding the resot of the locales ? Meaning zh, ko, jp ?

I contacted Rebecca, I'm sending over a file with these strings for translations.

> We also need to update the [default template](https://github.com/jhadvig/oauth-server/blob/CONSOLE-2535/pkg/server/login/templates.go) so it contains the go macros.
> Could you also link the oauth-template PR.
> THanks ! :)

Will update.

**Comment by jhadvig** [comment]
>Moved strings to locales.go
Is this correct?

yes, since it's just a handful of strings its better to hardcode them, so we dont have to read them on runtime

File: pkg/server/login/locales.go [hunk 1]
```diff
@@ -37,7 +37,12 @@ var supportedLangs = []language.Tag{
 }
 
 var locale_en = map[string]string{
-	"": "",
+	"LogInToYourAccount": "Log in to your account",
```
Comment (jhadvig) [inline pkg/server/login/locales.go:40]: could you please put these to 4 spaces, so it align with go's standard

**Review by jhadvig — COMMENTED** [review]
@sg00dwin one small comment, otherwise the PR looks good 👍 
Lets wait for rest of the locales and merge this :)

File: pkg/server/login/locales.go (hunk 1 above)
Comment (sg00dwin) [reply pkg/server/login/locales.go:40]: ok I ran `gofmt` on that file. Let me know if it's still not correct.

Comment (jhadvig) [reply pkg/server/login/locales.go:40]: looks fine 👍

**Review by jhadvig — APPROVED** [review]

**Pull request merged**